activesupport/lib/active_support/railtie.rb

```diff
@@ -13,6 +13,13 @@ class Railtie < Rails::Railtie # :nodoc:
       end
     end
 
+    initializer "active_support.halt_callback_chains_on_return_false", after: :load_config_initializers do |app|
```

Comment:
@rafaelfranca Brand new Rails 5 come [with an initializer file called `config/initializers/callback_terminator.rb`](https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/initializers/callback_terminator.rb) which sets the value for `app.config.active_support.halt_callback_chains_on_return_false`.

In this code, we check if that value has been set and eventually assign it to `ActiveSupport::Callbacks::CallbackChain.halt_and_display_warning_on_return_false`.

Therefore, it made sense to me to run this method **after** the `config/initializers/callback_terminator.rb` has been called.

What kind of issues is this causing?